### DISCUSSION TITLE
WTF!!!

### DISCUSSION THREAD

**Discussion by infinitegame111** [cefsharp/CefSharp#4129] [General]
What the f%$# is going on between 80 to the newest version! It glitches so bad it's half white and black and when I resize the window it crashes.

**Comment by psmulovics** [comment]
Let me help you how you should have said this.

@amaitland , thank you for the amazing work you have put into this open source project, free of charge and you spending so much time of helping people. I have the following problem, with this version, here is a repro project, etc

**Comment by amaitland** [comment]
> What the f%$# is going on between 80 to the newest version! It glitches so bad it's half white and black and when I resize the window it crashes.

Whilst there isn't an explicit `Code of Conduct` for this project, it goes without saying the language you have used is **unacceptable**.

If you require assistance then you'll need to do the following:

- Edit your `Title` and provide a few words to summarise the problem (no profanity).
- Edit your content to provide a detailed description of the problem, steps to reproduce, the **exact error** you are seeing.

https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following has some guidance on what to include when asking a question.

- Make sure to test with the `MinimalExample` to see if the same problem reproduces on your machine.
- Check the log file for errors https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file
